docs/src/conf.py

```diff
@@ -191,6 +191,14 @@ def autolog(message: str, section: str | None = None, color: str | None = None)
 tippy_enable_doitips = False
 tippy_rtd_urls = [
     "https://www.sphinx-doc.org/en/master/",
+    "https://geovista.readthedocs.io/en/latest/",
+    "https://geovista.readthedocs.io/en/stable/",
+    "http://localhost:11000",
+    "http://0.0.0.0:11000",
+    "https://matplotlib.org/stable/",
+    "https://numpy.org/doc/stable/",
+    "https://platformdirs.readthedocs.io/en/stable/",
+    "https://rasterio.readthedocs.io/en/stable/",
```

Comment:
These packages serve their documentation on RTD using the [/api/v3/embed/](https://docs.readthedocs.io/en/stable/api/v3.html#get--api-v3-embed-) protocol, which `sphinx-tippy` supports and thus can scrape such resources for tooltips.